### ISSUE TITLE
No inflection icon generated for κάτεισι  (Greek verb)

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#291]
double click on    κάτεισι     line 6  from  https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0096.tlg002.First1K-grc1:1-4b/
no inflection icon is generated.

the Morpheus output is the following:

<hdwd xml:lang="grc">κάτειμι</hdwd>
<pofs order="1">verb</pofs>
</dict>
<infl>
<term xml:lang="grc">
<stem>κατά:εισι</stem>
</term>
<pofs order="1">verb</pofs>
<mood>indicative</mood>
<num>singular</num>
<pers>3rd</pers>
<tense>present</tense>
<voice>active</voice>
<stemtype>irreg_mi</stemtype>
<morph>unasp_preverb</morph>
</infl>

are we missing the match because of   we don't have this table?

**Comment by balmas** [comment]
I believe we don't get a match here because our rule for the irreg_mi stemtype also specifies the headword εἰμί. And that is the verb 
It would be easy enough to add another rule that says *any* word which an irreg_mi stemtype should show the εἰμί table, or alternatively to add an additional match to the εἰμί table just for the headword κάτειμι. 

@monzug what do you think would be best?

**Comment by monzug** [comment]
I would rather go for the first solution. but I ma not so comfortable unless there is time for testing.  it would be nice to know how many irreg_mi stemtype we have in morpheus and see if the fix would work for a subset of these words. do you think it would be possible to have such a list?

**Comment by balmas** [comment]
I would have to look into the data to see if I can figure that out.  I think this won't be something we try to do right now anyway so I guess we can take our time.

**Comment by monzug** [comment]
agree. I did try variation of the Morpheus tool with no much luck...

**Comment by balmas** [comment]
see related item #287

**Comment by monzug** [comment]
if we add the headword κάτειμι to rules.csv, we should also add the compound of ἵημι  for table verbpdgm34 such as ἀφίημι

**Comment by monzug** [comment]
btw, line 76 in rules.csv is a dup of line 75.
right now we have two lines like this
verbpdgm35	3	verb		middle		aorist	ἵημι
we should change middle in mediopassive in one of the two identical lines.
